setupTests-integration.js

```diff
@@ -7,6 +7,28 @@ import {
   precalculationHelpers, WalletPrecalculationHelper,
 } from './scripts/helpers/wallet-precalculation.helper';
 
+expect.extend({
+  toBeInArray(received, expected) {
```

Comment:
This custom matcher was added to test that an address is on the known wallet addresses array.
We cannot use the usual `expect(array).toContain(element)` matcher since the place we need this is inside an object matcher (`expect(obj).toMatchObject(...)`)